am-ui/src/app/components/home/home.component.ts

```diff
@@ -1,10 +1,25 @@
-import {Component} from '@angular/core';
+import {TranslateService} from '@ngx-translate/core';
+import {Component, OnInit} from '@angular/core';
+import {Subscription} from 'rxjs';
 
 @Component({
   selector: 'mp-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss'],
 })
-export class HomeComponent {
-  constructor() {}
+export class HomeComponent implements OnInit {
+  public appLanguage: string;
+  public subscription: Subscription;
+
+  constructor(private translate: TranslateService) {}
+
+  ngOnInit(): void {
+    this.subscription = this.translate.onLangChange.subscribe(appLanguage => {
+      this.appLanguage = appLanguage.lang;
+    });
+  }
+
+  ngOnDestroy() {
```

Comment:
Might as well implement the OnDestroy interface here (similar to how it already implements OnInit) to get rid of this warning

Reply:
True, also I think might not go correctly through life cycle if not done under implements